src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/TaskTestExtensions.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Threading.Tasks;
+using Xunit.Abstractions;
+
+namespace Microsoft.Diagnostics.Monitoring.TestCommon
+{
+    public static class TaskTestExtensions
+    {
+        public static async Task SafeAwait(this Task task, ITestOutputHelper outputHelper = null)
```

Comment:
These methods did not change but were split out into their own extension method class.